packages/common/src/core/types/IHubEntityItemBase.ts

```diff
@@ -0,0 +1,42 @@
+import { IGeometry } from "@esri/arcgis-rest-types";
+import { IHubEntityBase } from "./IHubEntityBase";
+
+/**
+ * Properties exposed by Entities that are backed by Items
+ */
+export interface IHubEntityItemBase extends IHubEntityBase {
+  /**
+   * Thumbnail Uril (read-only)
+   */
+  readonly thumbnailUrl: string;
```

Comment:
I'm not sure re: events, and discussions definitely do not. Let's leave until we implement something that needs it. It's also not awful to put it right on the `IHubUser` interface either. Since this is not implementation inheritance, in the end, it really does not matter that much how it gets there